javascript/modules/client/test/integration/client-addressList.test.ts

```diff
@@ -36,7 +39,6 @@ const IPFS_API_KEY = process.env.IPFS_API_KEY ||
 const web3endpoints = {
   working: [
     "https://mainnet.infura.io/v3/94d2e8caf1bc4c4884af830d96f927ca",
-    "https://cloudflare-eth.com/",
```

Comment:
Don't we have another working one? Otherwise, `shiftNode` tests are less representative